packages/mask/src/extension/popups/pages/Personas/ConnectedWallets/index.tsx

```diff
@@ -46,14 +46,24 @@ const ConnectedWallets = memo(() => {
 
                     return {
                         ...x,
-                        name: `${x.platform} wallet ${index + 1}`,
+                        name: '',
                     }
                 }
                 return null
             }),
         )
 
-        return compact(results)
+        return sortBy(compact(results), (x) => Number(x.created_at))
+            .reverse()
```

Comment:
```suggestion
        return sortBy(compact(results), (x) => -Number(x.created_at))
```

Reply:
The code has been changed. It need reverse after `map`

Reply:
sort with `-Number(x.created_at)` is  to `reverse`